next/components/search/search-multicheckbox-facet.tsx

```diff
@@ -52,18 +52,21 @@ export function MultiCheckboxFacet({
                   )}`}
                   checked={checked}
                   onClick={() => (checked ? onRemove(value) : onSelect(value))}
+                  aria-label={`example_facet_${label}${getFilterValueDisplay(
+                    option.value
+                  )}-title`}
                 />
-                <label
+                <span
                   className="ml-2 text-sm text-steelgray"
-                  htmlFor={`example_facet_${label}${getFilterValueDisplay(
+                  title={`Filter by ${getFilterValueDisplay(
```

Comment:
We should translate this?

Reply:
That is a brilliant question. I feel that we could decide on static label which will be translated for each local. For example: Filter by, tags etc